aggregate/housing_production/area_within_historic_district.py

```diff
@@ -72,7 +74,9 @@ def fraction_PUMA_historic(PUMA, hd):
 
 
 def load_historic_districts_gdf() -> gp.GeoDataFrame:
-    hd = gp.read_file(".library/lpc_historic_district_areas.csv")
+    df = read_from_S3("lpc_historic_district_areas")
```

Comment:
Nice to have the `read_from_S3` function to standardize datasets being uploaded via the dataloading step from data library